src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -3,78 +3,104 @@
+                    <div class="row mt-1">
+                        <div class="col">
+                            <div
+                                v-show="drawingStateMessage"
+                                class="d-flex justify-content-center drawing-toolbox-drawing-state"
+                                :class="{ 'text-danger': isDrawingStateError }"
+                            >
+                                {{ drawingStateMessage }}
+                            </div>
+                        </div>
                     </div>
-                    <div class="d-flex justify-content-center">
-                        <button
-                            :disabled="isDrawingEmpty"
-                            class="btn-light btn m-1"
-                            data-cy="drawing-toolbox-delete-button"
-                            @click="showClearConfirmation"
-                        >
-                            {{ $t('delete') }}
-                        </button>
-                        <DrawingExporter :is-drawing-empty="isDrawingEmpty" />
-                        <button
-                            type="button"
-                            class="btn btn-light m-1"
-                            :disabled="isDrawingEmpty || !kmlLayerId"
-                            data-cy="drawing-toolbox-share-button"
-                            @click="openShare"
-                        >
-                            {{ $t('share') }}
-                        </button>
+                    <div class="row g-2">
+                        <div class="col d-grid">
+                            <button
+                                :disabled="isDrawingEmpty"
+                                class="btn-light btn"
+                                data-cy="drawing-toolbox-delete-button"
+                                @click="showClearConfirmation"
+                            >
+                                {{ $t('delete') }}
+                            </button>
+                        </div>
+                        <div class="col d-grid">
+                            <DrawingExporter :is-drawing-empty="isDrawingEmpty" />
+                        </div>
+                        <div class="col d-grid">
+                            <button
+                                type="button"
+                                class="btn btn-light"
+                                :disabled="isDrawingEmpty || !kmlLayerId"
+                                data-cy="drawing-toolbox-share-button"
+                                @click="openShare"
+                            >
+                                {{ $t('share') }}
+                            </button>
+                        </div>
+                    </div>
+                    <div v-if="isDrawingLineOrMeasure" class="row mt-2">
+                        <div class="col d-grid">
+                            <button
+                                v-if="isDrawingLineOrMeasure"
```

Comment:
Is this `v-if` needed if already in line 73

Reply:
no it is not required anymore... good catch